contracts/Example1.sol

```diff
@@ -1,7 +1,8 @@
+//SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.4;
 
 contract Example1 {
     uint8 public day = 10;
-    uint256 public amount = 1000;
+    uint16 public amount = 1000;
```

Comment:
uint type of amount should not be changed since the amount can easily overflow uint16. The expectation was to reorder and have this order which would require two slots
```
uint8 public day;
address public ownerAddr;
uint256 public amount;
```